### DISCUSSION TITLE
[FEATURE] Base64 Based Challenge

### DISCUSSION THREAD

**Discussion by aashu10sh** [lennis-dev/hackit#31] [Ideas]
### Is there an existing discussion for this?

- [X] I have searched the existing discussions

### Describe your idea

I was thinking of a challenge where we can introduce the users to base64 encoding. It would be similar to the 'easy-beginning' challenge, here they'd have to look at the JavaScript source code and decode the base64 being compared ( to users input)  to find the correct solution. 

More Detail:
Name : "Basic Challenge"
Description: " I heard there's a technique for encoding files safely in the web, something 64?"

### Anything else?

This is my first ever open source contribution ( attempt ). I hope you understood what I am trying to say:)

**Comment by lennis-dev** [comment]
Good idea I have opened an issue here #32
